app/facades/items_facade.rb

```diff
@@ -0,0 +1,15 @@
+class ItemsFacade
+  attr_reader :service
+
+  def initialize
+    @service = start_service
+  end
+
+  def get_item(id)
+    item = Item.new(service.item(id))
+  end
+
+  def start_service
+    @service = EngineService.new
+  end
```

Comment:
Instead of initializing and using an attr_reader, I might instead do something like:

```ruby
def service
  @service ||= EngineService.new
end
```